spec/requests/invites_controller_spec.rb

```diff
@@ -203,6 +203,52 @@
     end
   end
 
+  context '#create_multiple_use_invite_link' do
```

Comment:
```suggestion
  context 'create multiple use invite link' do
```

`#` is meant to represent an instance method